package.json

```diff
@@ -6,6 +6,8 @@
   "license": "Apache 2.0",
   "repository": "https://github.com/oppiabot/oppiabot.git",
   "scripts": {
+    "dev": "nodemon",
+    "lint": "standard --fix",
```

Comment:
It is so that any auto-fixable issues found during the lint will be fixed automatically

Reply:
Just as a side note, I noticed that the rules set by standard and different and they go against themselves

Reply:
> I noticed that the rules set by standard and different and they go against themselves

Can you rephrase this?

> It is so that any auto-fixable issues found during the lint will be fixed automatically

Don't do this else CI (in future) will fix the issue in the local repo and it will assume that there isn't any (lint) issue in PR!

Reply:
> Don't do this else CI (in future) will fix the issue in the local repo and it will assume that there isn't any (lint) issue in PR!

Noted.

> Can you rephrase this?

I meant the linting rules set by `standard` contradicts those set by `eslint`